FileAPI/fileReader.any.js

```diff
@@ -0,0 +1,59 @@
+// META: title=FileReader States
+
+'use strict';
+
+test(function () {
+  assert_true(
+    "FileReader" in window,
```

Comment:
This test doesn't make sense in a .any.js file. Probably just getting rid of the test entirely makes sense since the idlharness tests already test this.

Reply:
Updated to use `globalThis` instead of `window`.